.github/workflows/release-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Release Publisher
+
+on:
+  push:
+    tags:
+      - "*.*.*"
+
+permissions:
+  contents: read
```

Comment:
It's fine that this is here, just curious if it _needs_ to be specified for some reason

Reply:
This is a good point. I just copy/pasted this from the examples on the Release Drafter repo. It may work without this, but I haven't tested that.